website/docs/r/sql_server.html.markdown

```diff
@@ -80,7 +97,19 @@ The following attributes are exported:
 
 -> You can access the Principal ID via `${azurerm_sql_server.example.identity.0.principal_id}` and the Tenant ID via `${azurerm_sql_server.example.identity.0.tenant_id}`
 
-## Timeouts
+---
+
+A `extended_auditing_policy` block supports the following:
+
+* `storage_account_access_key` - (Required)  Specifies the access key to use for the auditing storage account.
+
+* `storage_endpoint` - (Required) Specifies the blob storage endpoint (e.g. https://MyAccount.blob.core.windows.net).
+
+* `storage_account_access_key_is_secondary` - (Optional) Specifies whether `storage_account_access_key` value is the storage's secondary key.
```

Comment:
could we order these alphabetically?

Reply:
So we don't need to put `required` fields in advance? We just order all `required` and `optional` fields alphabetically, right?

Reply:
WHoops missed the optional, yes typically alphabetically, but sometimes i'll order for aesthetics ie `storage_endpoint` -> `storage_account_access_key` -> `storage_account_access_key_is_secondary`